crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2217,6 +2335,166 @@ swwwwwwwwwwwwwwww
 swwwwwwwwwwwwww
 ENDMAP
 
+NAME:  nicolae_shoals_bring_sunscreen
+TAGS:  transparent water_ok no_tide
+DEPTH: Shoals:2-
+KMONS: S = sun demon / sun moth
```

Comment:
imo this is too many sun moths; they're a top threat in a different sbranch and this vault placed 4-5 of them when I was testing it. Maybe nsubst in like one of them and do sun demons plus branch native monsters? Cool theme though.

Reply:
NSUBST now places 1-3 sun moths, with 1 guaranteed and 1/3 chance of each of the other two.